docs/docs/reference/crd-reference/appcontext.md

```diff
@@ -0,0 +1,125 @@
+---
+comments: true
+---
+
+# KeptnAppContext
+
+(Need to add description)
+
+It works with the corresponding
+[KeptnApp](app.md) resource
+that has the same name and is located in the same namespace.
+
+`KeptnAppContext` lists the tasks and evaluations
+to be executed pre/post-deployment.
+Tasks referenced by `KeptnAppContext` are defined in a
+[KeptnTaskDefinition](taskdefinition.md)
+resource.
+`KeptnAppContext` identifies the task by the value of the `metadata.name` field
+and does not need to understand what runner is used to define the task.
+Similarly, evaluations referenced are defined in a
+[KeptnEvaluationDefinition](evaluationdefinition.md)
+resource and identified by the value of the `metadata.name` field;
+`KeptnAppContext` does not need to understand
+the data source or query being used for the evaluation.
+
+## Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1beta1
+kind: KeptnAppContext
+metadata:
+  name: <app-name>
+  namespace: <app-namespace>
+spec:
```

Comment:
We are missing metadata and spanLinks here

Reply:
Fixed